BismarkUnchained/scenes/Main/Main.gd

```diff
@@ -1,6 +1,7 @@
 extends Node2D
 
 var time_delta = 1
+var time_elapsed = 0
```

Comment:
maybe document what this is? it looks like elapsed game time b/c it takes in `time_delta`